modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/quantity_selector/InputQuantitySelector.js

```diff
@@ -19,15 +19,29 @@ import {
 } from '../../utilities/quantities';
 import RulesPopover from './RulesPopover';
 
-const getErrors = (value, min, max, step, precision = 0) => {
+const getErrors = (
+	value,
```

Comment:
[NB]: arguments have become many, being local we could consider converting from separate function arguments to a single object.